INSTALL.md

```diff
@@ -26,12 +26,15 @@ There are several other distributions of the PureScript compiler available, whic
 * NPM: `npm install -g purescript`
 * Homebrew (for macOS): `brew install purescript`
 * [PSVM](https://github.com/ThomasCrevoisier/psvm-js): `npm install -g psvm`
+* [Hackage](https://hackage.haskell.org/package/purescript): `cabal install purescript` or `stack install purescript`
```

Comment:
This will build a compiler binary with different dependencies than what we released, so this is _not_ a supported way of obtaining compiler binaries and we don't want to point people in that direction. It causes weird issues we can't reproduce.

Reply:
Getting the compiler from Hackage is just compiling from source so I’d rather not add it to this list anyway, as compiling from source is covered separately.